### ISSUE TITLE
HOST for integrating Shelly into Home Assistant

### ISSUE THREAD

**Issue by stirls1** [home-assistant/frontend#15299]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Hi, when I browse to Home Assistant on my PC, it gets to the Pi fine. I try to add Integration for Shelly but it requires a 'Host*'. Nothing in the instructions explains what I should enter for the Shelly 'Host*'. The 'help' basically says 'just enter whatever you need'.   What should I enter as the 'Host*' please?

### Describe the behavior you expected

When I browse to Home Assistant on my PC and try to add an Integration for Shelly, it should automatically find the Shelly devices and should not ask me for a  'Host*'.  I think the issue may be related to my network infrastructure. The Pi that Home Assistant is running on is at 192.168.1.119 but my Shelly devices are getting IPs from 192.168.0.1 router. 

### Steps to reproduce the issue

1. On my Windows 10 PC, browse to Home Assistant homeassistant.loca.:8123
    It opens the Home Assistant screen
2. Click 'Settings'
3. Click 'Devices & Services'
4. Click 'Add Integration'
5. Search for 'Shelly' and click the Shelly icon
6. Home assistant requires a 'Host*' and fails to integrate Shelly.

### What version of Home Assistant Core has the issue?

Home Assistant 2023.1.7

### What was the last working version of Home Assistant Core?

N/A

### In which browser are you experiencing the issue with?

Chrome and Internet Explorer

### Which operating system are you using to run this browser?

Windows 10

### State of relevant entities

```txt
PC and Pi are on 192.168.1.* network whereas Shelly and Tuya devices are on 192.168.0.*
```

**Comment by schraubgern** [comment]
I've done the same as Stirls1:
I have updated to the latest available Home Assistant version.
I have cleared the cache of my browser.
I have tried a different browser to see if it is related to my browser.

I have today updated to version 2023.2.7 from Home Assistant 2023.1.6 Supervisor 2023.01.1 Operating System 9.5 Frontend 20230110.0 - latest 
My Issue: after the update then my Shelly Integration doesn't work with my Shelly dimmer anymore. 
After restore to my old SW-Release  Home Assistant 2023.1.6 Supervisor 2023.01.1 Operating System 9.5 Frontend 20230110.0 - latest all things works as aspected as before.

**Comment by stirls1** [comment]
The solution I ended up with was to change/simplify my network topology.

Initially, I had a DHCP router for 192.168.0.* and another router under it that also did DHCP for 192.168.1.*.
I turned off the DHCP of one router, and everything merged to the 192.168.1.* subnet (some automatically, anything with static IPs I had to do manually).
Once that was sorted, Home Assistant no longer asked for a 'Host' when trying to integrate Shelly.
